### DISCUSSION TITLE
Content of demo site missing

### DISCUSSION THREAD

**Discussion by hughlilly** [thegetty/quire#195] [Feedback]
Hello,

I can load https://thegetty.github.io/quire-starter/ fine, but the content that ought to be at https://thegetty.github.io/quire-starter/contents does not appear to be there.

I can't find the repo this site lives in, else I would have had a look and put in a PR there.

Thanks
Hugh

**Comment by geealbers** [comment]
Thanks for this catch @hughlilly. We're going to need to redeploy the demo site. The page that you can see is a holdover from a time when Quire was in three separate repos. I think the fact that you can see the main landing page at all is from GitHub caching as there's no longer a proper repo behind that site.

@Erin-Cecele, I think the best way for us to do this going forward, is to use Netlify Drop to put a site at https://quire-demo.netlify.app (which I just did), and then maybe also do a redirect so we can have a non-netlify URL to share? Something https://quire.getty.edu/demo. What do you think?

**Reply by Erin-Cecele** [reply]
@geealbers I think that's a great idea. Do you think we should link to that site in the docs anywhere we reference the default starter content? Is there anywhere else we should consider linking to the demo site?

**Comment by hughlilly** [comment]
Perhaps it's worth noting here too that https://gettypubs.github.io/quire/, which is linked to from the colophon on [this Getty publication](https://www.getty.edu/interns-2020/report/foreword/), among probably a number of others, redirects to https://quire.netlify.app instead of https://quire.getty.edu

**Reply by geealbers** [reply]
Thanks again @hughlilly for the nice catches! We'll get those updated too.
